.github/ISSUE_TEMPLATE/refactor_tickets.md

```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
```

Comment:
```suggestion
name: 🔧 Refactor ticket
```

may I suggest a different icon, to make it easier to distinguish between the issue types? :wink: